internal/route/repo/view.go

```diff
@@ -132,6 +132,7 @@ func renderFile(c *context.Context, entry *git.TreeEntry, treeLink, rawLink stri
 		return
 	}
 
+	c.Data["Title"] = blob.Name() + " - " + c.Data["Title"]
```

Comment:
see [macaron.Context.Data](https://github.com/go-macaron/macaron/blob/b2cd309dc711ad238e63785012080a37ccd6314c/context.go#L97), the type is ```map[string]interface{}```, it should be change to ```string```.